Lib/netrc.py

```diff
@@ -114,17 +118,18 @@ def _parse(self, file, fp, default_netrc):
                     "bad toplevel token %r" % tt, file, lexer.lineno)
 
             if not entryname:
-                raise NetrcParseError("missing %r name" % tt, file, lexer.lineno)
+                raise NetrcParseError(
+                    "missing %r name" % tt, file, lexer.lineno)
```

Comment:
looks like an irrelevant formatting change
```suggestion
                raise NetrcParseError("missing %r name" % tt, file, lexer.lineno)
```